### ISSUE TITLE
Namecoin / Onename verification

### ISSUE THREAD

**Issue by noflag** [keybase/keybase-issues#1511]
Could you add namecoin verification, or publishing keybase data onto namecoin, similar to how Onename does it?

**Comment by zQueal** [comment]
Duplicate of #518.

**Comment by sebie** [comment]
Yes, it partially is a duplicate @zQueal , but I think that he is also asking whether or not
> publishing keybase data onto namecoin

would be possible.

-----

And @noflag I think they use dnschain.
